policy/modules/contrib/boothd.te

```diff
@@ -0,0 +1,82 @@
+policy_module(boothd, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type boothd_t;
+type boothd_exec_t;
+init_daemon_domain(boothd_t, boothd_exec_t)
+
+type boothd_etc_t;
+files_config_file(boothd_etc_t)
+
+type boothd_unit_file_t;
+systemd_unit_file(boothd_unit_file_t)
+
+type boothd_var_run_t;
+files_pid_file(boothd_var_run_t)
+
+type boothd_var_t;
+files_type(boothd_var_t)
+
+########################################
+#
+# boothd local policy
+#
+allow boothd_t self:fifo_file rw_fifo_file_perms;
+allow boothd_t self:capability { chown dac_override dac_read_search net_admin setgid setuid sys_nice sys_resource };
```

Comment:
The domain requires quite a lot of powerful capabilities, was it justified sufficiently?

Reply:
Dac_override is required because the /etc/booth/booth.key file must be readable by both hacluster and root and it's not possible to change permission to root.
net_admin is brought by setsockopt syscall which is caused by sd_notifyf